app/views/forms/_comments_form.html.erb

```diff
@@ -0,0 +1,10 @@
+
+<%= form_with model: [post, comment] do |form| %>
+  <div class="mb-3">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: "form-control" %>
+  </div>
+  <div>
+    <%= form.submit class: "btn btn-outline-secondary" %>
+  </div>
```

Comment:
- [ ] Please the should be a form in the views/comments/new.html.erb file which accepts input for creating new comments.